workspace/notebook/py_unit_tests_functions.json

```diff
@@ -469,7 +469,7 @@
 					}
 				},
 				"source": [
```

Comment:
The function test_hrm_to_curated_no_dropping_primary_ids_only_records is being used in all unit test notebooks and removing the default value might break other notebooks. Can I please ask why it's been removed and if can be put back?

Reply:
I have added default value test_hrm_to_curated_no_dropping_primary_ids_only_records  function